### DISCUSSION TITLE
Browserslist: caniuse-lite is outdated

### DISCUSSION THREAD

**Discussion by josephmo** [tailwindlabs/tailwindcss#9897] [Help]
I recently started seeing this message when I do a push to the production, test and staging server:

`Browserslist: caniuse-lite is outdated`

This is right after I do:

`rails tailwindcss:build`

Then I see the following message:

```
+ /home/deploy/.rbenv/versions/3.1.0/lib/ruby/gems/3.1.0/gems/tailwindcss-rails-2.0.10-x86_64-linux/exe/x86_64-linux/tailwindcss -i /home/deploy/current/app/assets/stylesheets/application.tailwind.css -o /home/deploy/current/app/assets/builds/tailwind.css -c /home/deploy/current/config/tailwind.config.js --minify
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
```

I did not install caniuse-lite  in the first place.

**Comment by viniciusbavosa** [comment]
i'm having the same problema but is when i do " npx tailwindcss -i ./tailwind/input.css -o ./src/styles/output.css --watch"

**Reply by josephmo** [reply]
This is before you do: rails tailwindcss:build?

**Reply by viniciusbavosa** [reply]
No, its after. i'm using npm, so everyday i open my project and start tailwind with the command above, and then the error message pops up... i do think it's affecting tailwind compiler tho, today i had a problem with styling and my friend who helped me said that my compiled css code was wrong

**Comment by Linkinlog** [comment]
Im not using rails yet am also having the same issue with the standalone executable, im running `tailwindcss --input web/input.css --output assets/app.css`

**Comment by kangkyu** [comment]
I think this issue has been fixed after this bug report. The message is gone when I did `bundle update tailwindcss-rails`
